pkg/routeagent_driver/constants/constants.go

```diff
@@ -36,7 +36,7 @@ const (
 	// part of route-agent and this will subsequently be used in globalnet controller for
 	// supporting connectivity from HostNetwork to remoteClusters.
 	// [#] interface on the node that has an IPAddress from the clusterCIDR
-	CniInterfaceIP = "submariner.io/cniIfaceIp"
+	CniInterfaceIP = "submariner.io/cniIfaceIP"
```

Comment:
Changing the label name may have other effects and would be outside the scope of this PR.

```suggestion
	CniInterfaceIP = "submariner.io/cniIfaceIp"
```